Engine/DataFeeds/Subscription.cs

```diff
@@ -32,11 +33,19 @@ public class Subscription : IEnumerator<SubscriptionData>
     {
         private bool _removedFromUniverse;
         private readonly IEnumerator<SubscriptionData> _enumerator;
+        private List<SubscriptionRequest> _subscriptionRequests;
 
         /// <summary>
         /// Gets the universe for this subscription
         /// </summary>
-        public Universe Universe { get; }
+        public IEnumerable<Universe> Universe => _subscriptionRequests
+            .Where(x => x.Universe != null)
```

Comment:
as a note -- the only case of a null universe should be subscriptions added for the sole purpose of securities added via `EnsureCurrencyDataFeeds`